packages/passport-interface/src/zuconnect.ts

```diff
@@ -0,0 +1,62 @@
+import { ZuzaluUserRole } from "./zuzalu";
```

Comment:
I think I've finished shuffling these constants around now. This ought to be their final resting place.